src/qt/bitmap.cpp

```diff
@@ -409,25 +410,29 @@ void wxBitmap::SetDepth(int depth)
 void *wxBitmap::GetRawData(wxPixelDataBase& data, int bpp)
 {
     void* bits = NULL;
+
+    wxBitmapRefData *bitmapRefData = static_cast<wxBitmapRefData *>(m_refData);
```

Comment:
This is very minor, but `bitmapRefData` just seems so verbose... the existing code mostly calls it `refData`, so please feel free to shorten it if you'd like. If you prefer the longer name, please also free to leave it like this.

Reply:
Thanks for the feedback,  I've applied all these suggestions.